python/wawenets/demo.py

```diff
@@ -0,0 +1,286 @@
+import subprocess
+import sys
+
```

Comment:
it would be nice if they were totally exact! but in fact I had to relax the precision on one of the tests because the values were different on the two platforms I tested (intel/macOS and intel/ubuntu). I'm quite happy with differences that small to be honest.

I'll add a print statement somewhere saying that the values should be quite close but perhaps not exact.